interview/.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+.vscode
```

Comment:
I wonder if here should be implemented also **/*.pyc?

Reply:
No, these files are ignored by nested .gitignore.

Reply:
No, this is handled by the nested .gitignore.